### PR TITLE
Fixes/Updates during Windows regression testing

### PR DESCRIPTION
**Pull request by mikefero**
NOTE: The beta v5 protocol is currently broken due to changes that are
going into the official v5 protocol on the Cassandra v4.0 development
work. In this PR is a commit to ensure that the result_metadata_id is not
handled.

https://issues.apache.org/jira/browse/CASSANDRA-10786
https://www.mail-archive.com/dev@cassandra.apache.org/msg11731.html

### CODE REVIEW

File: src/protocol.hpp [hunk 1]
```diff
@@ -27,7 +27,13 @@ inline bool supports_set_keyspace(int version) {
 }
 
 inline bool supports_result_metadata_id(int version) {
-  return version >= CASS_PROTOCOL_VERSION_V5;
+  return false;
+  // Cassandra v4.x broke the beta protocol v5 version
+  // JIRA: https://issues.apache.org/jira/browse/CASSANDRA-10786
+  // Mailing List Discussion: https://www.mail-archive.com/dev@cassandra.apache.org/msg11731.html
+  // TODO: Remove after Cassandra v4.0.0 is released
+  return false;
```
Comment (mpenick) [inline src/protocol.hpp]: Returning false twice.

**Comment by mpenick** [comment]
Remember to merge this into your other PR first. :)

**Comment by mikefero** [comment]
Landed 125e452

File: src/protocol.hpp (hunk 1 above)
Comment (mikefero) [reply src/protocol.hpp]: Corrected and merged with base branch.

**Pull request closed without merging**